### PR TITLE
add example for getting event details and registering wallet

### PR DESCRIPTION
**Pull request by kristiantokarim**
Add example for get wallet list details sdk function and register wallet functions

https://user-images.githubusercontent.com/12123712/187953450-1f0474c9-6be4-47ca-b4a6-b55ff47e54a5.mov

### CODE REVIEW

File: register-wallet-to-event/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      const shippingStatus = await sdk.getShippingInfoStatus(itemStockId.value)
+      if (!shippingStatus) {
+        throw new Error('can not fetch shipping status : ', shippingStatus)
+      }
+      const shippingInfo = await sdk.getShippingInfo(itemStockId.value)
+      if (!shippingInfo) {
+        throw new Error('can not fetch shipping info : ', shippingInfo)
+      }
+
+      const firstName = document.querySelector<HTMLInputElement>('#firstName')!
```
Comment (tomonari-t) [inline register-wallet-to-event/src/main.ts]: Have you used these ?

**Review by tomonari-t — CHANGES_REQUESTED** [review]

File: register-wallet-to-event/src/main.ts (hunk 1 above)
Comment (kristiantokarim) [reply register-wallet-to-event/src/main.ts]: Removed

File: register-wallet-to-event/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Register wallet to event example
+
+This example shows how to get event information with otp and register wallet address to event
+
+To start this, install the modules and try out with own ACCESS TOKEN and Fortmatic key.
```
Comment (tomonari-t) [inline register-wallet-to-event/README.md]: We have not yet supported Fortmatic so no need to get Fortmatic key

File: register-wallet-to-event/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## How to use
```
Comment (tomonari-t) [inline register-wallet-to-event/README.md:7]: And clients do not know how to get event id so add how to do that simply

**Review by tomonari-t — APPROVED** [review]

**Pull request merged**